dockstore-webservice/src/main/java/io/dockstore/webservice/core/Image.java

```diff
@@ -153,4 +163,19 @@ public void setOs(final String os) {
         this.os = os;
     }
 
+    public Timestamp getDbUpdateDate() {
+        return dbUpdateDate;
+    }
+
+    public long getSize() {
```

Comment:
Thought: Wonder if it should be a `Long` as an instance variable when it is get and set as a `long`?
Alternatively, should it be `public Optional<Long> getSize()` to deal with when the instance variable is set to null?

Reply:
Ya, both the getter and setter are supposed to be Long. Shouldn't be Optional since Long can handle null just fine. We have an overruse of Optional according to https://stackoverflow.com/a/26328555  . I find that we often go "Can it be null? Oh, it can. Let's use Optional." instead of a simple null check.